subscription/src/main/java/org/killbill/billing/subscription/api/user/DefaultSubscriptionBaseApiService.java

```diff
@@ -279,6 +281,7 @@ private boolean doCancelPlan(final Map<DefaultSubscriptionBase, DateTime> subscr
                     throw new SubscriptionBaseApiException(ErrorCode.SUB_CANCEL_BAD_STATE, subscription.getId(), "PENDING CANCELLED");
                 }
                 // Similarly, if subscription is cancelled with date past the expiry date (in case of a FIXEDTERM phase), we disallow the operation
+                // FIXME-MERGE-MASTER-TO-WFR-023x: Is this should be exist?
```

Comment:
I am not sure I understand what is meant here?

Reply:
I actually asking whether these lines:

```java
  if (pendingTransition != null &&
    pendingTransition.getTransitionType() == SubscriptionBaseTransitionType.EXPIRED &&
    pendingTransition.getEffectiveTransitionTime().compareTo(effectiveDate) < 0) {
      throw new SubscriptionBaseApiException(ErrorCode.SUB_CANCEL_BAD_STATE, subscription.getId(), "PENDING EXPIRY");
  }
```

should be removed or keep them exist. Because the diff from you and Reshma is different, and I'm not quite sure which one that is correct one. But I think is should be there, and thus I'll remove the `FIXME` line.